packages/shared/lib/sdk/sync.ts

```diff
@@ -248,6 +249,15 @@ interface TableauCredentials extends CredentialsCommon {
     token?: string;
     expires_at?: Date | undefined;
 }
+interface JWTCredentials {
```

Comment:
we have dont `TbaCredentials` so if we follow the same logic it should be `JwtCredentials`. Doesn't look as good but at least it is consistent

Reply:
Yes, makes sense to make it uniform, thanks , updated with the latest [commit](https://github.com/NangoHQ/nango/pull/2840/commits/6c49f1085afcc716d7d9622373faace0827f0e78)